File: QuanLyPhongKham.Server/src/QuanLyPhongKham.Models/Entities/DANHMUCDICHVU.cs
using MayNghien.Infrastructures.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyPhongKham.Models.Entities
{
    public class DANHMUCDICHVU : BaseEntity
    {
        [Key]
        public Guid MaDV { get; set; }
        public string TenDV { get; set; }
        public decimal DonGia { get; set; }
    }
}

File: QuanLyPhongKham.Client/src/models/DanhMucDichVu.js
/**
 * @typedef {Object} DanhMucDichVu
 * @property {string} maDV
 * @property {string} tenDV
 * @property {number} donGia
 */

export const createDanhMucDichVu = (
  maDV = "",
  tenDV = "",
  donGia = 0,
) => ({
  maDV,
  tenDV,
  donGia,
});

export const normalizeDanhMucDichVu = (item) => ({
  maDV: item?.maDV ?? item?.MaDV ?? "",
  tenDV: item?.tenDV ?? item?.TenDV ?? "",
  donGia: item?.donGia ?? item?.DonGia ?? 0,
});
